integrations/coinbase-integration/src/main/java/org/dash/wallet/integration/coinbase_integration/ui/dialogs/CoinBaseErrorDialog.kt

```diff
@@ -52,11 +58,34 @@ class CoinBaseErrorDialog : DialogFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
-        setOrHideIfEmpty(binding.coinbaseDialogTitle, "title")
-        setOrHideIfEmpty(binding.coinbaseDialogMessage, "message")
-        setOrHideIfEmpty(binding.coinbaseDialogIcon, "image")
-        setOrHideIfEmpty(binding.coinbaseDialogPositiveButton, "positive_text")
-        setOrHideIfEmpty(binding.coinbaseDialogNegativeButton, "negative_text")
+        lockScreenViewModel.activatingLockScreen.observe(this){
+            Log.e(this::class.java.simpleName, "Closing dialog")
```

Comment:
Do we really need these error logs in the logcat? I understand if it is used for development purposes, but it should be cleaned up afterward. Having random red messages in logcat that are not actually errors is confusing.